### ISSUE TITLE
External zsh-theme does not load correctly with antigen

### ISSUE THREAD

**Issue by edqu3** [zsh-users/antigen#283]
I want to use theme: [zeta](https://github.com/skylerlee/zeta-zsh-theme)
So I put in my _.zhrc_
`antigen theme skylerlee/zeta-zsh-theme zeta`

however the colors in the prompt are not loaded, ie. they default to grey.

This only happens when I have `ZSH_THEME="zeta"` commented out. 
Though this shouldn't be an issue since `antigen theme ... ` should have control over changing themes in zsh now.

``` shell
export ZSH=/home/xank/.oh-my-zsh
ZSH_THEME="zeta"   # removing this line results in colors not loading
source $ZSH/oh-my-zsh.sh
source /home/xank/antigen/antigen.zsh
antigen use oh-my-zsh
antigen bundle git
antigen bundle command-not-found
antigen bundle zsh-users/zsh-syntax-highlighting
antigen theme skylerlee/zeta-zsh-theme zeta    # however removing this line and only having the ZSH_THEME above set, results in a colored and colorless double prompt.
antigen apply
```
This only when I try to use external themes of the form `antigen theme user\repo theme-name`
So in order for the theme to load correctly, both lines must be included.

**Comment by desyncr** [comment]
@edqu3 Hello there. This is a known issue with 'locals' inside themes. It should be resolved in 1.2.1. See  https://github.com/zsh-users/antigen/issues/122

**Comment by desyncr** [comment]
@edqu3 You should be able to use zeta theme as expected with https://github.com/zsh-users/antigen/pull/284 `bugfix/hack-local-variables-in-themes` branch. Next to be merged in [1.2.1](https://github.com/zsh-users/antigen/milestone/10) release.

BTW you don't need to source `oh-my-zsh.sh` to be able to use OMZ's plugins and themes with Antigen,  with `antigen use oh-my-zsh` is enough.
